ansible_builder/main.py

```diff
@@ -307,6 +317,19 @@ def prepare_system_steps(self, bindep_output):
 
         return self.steps
 
+    def prepare_final_stage_steps(self):
+        self.steps.append("FROM {0}".format(self.base_image))
+        return self.steps
+
+    def prepare_galaxy_copy_steps(self):
+        if self.definition.get_dep_abs_path('galaxy'):
+            self.steps.extend(GalaxyCopySteps())
+        return self.steps
+
+    def delete_intermediate_image(self):
```

Comment:
Not sure it makes sense to have this live in the `Containerfile` class. Maybe this should live somewhere in `AnsibleBuilder`?

Reply:
Moving it to the bottom of AnsibleBuilder class for now.